include/metal/lambda/arg.hpp

```diff
@@ -19,7 +19,7 @@ namespace metal {
     /// A parametric \lambda that selects the n-th argument it is invoked with.
```

Comment:
I assume that you intentionally use n-th vs. (see below) `m`.

I think it is alright now, but I just want to make sure you did not miss it.

Reply:
I thought about changing it to `m-th`. Since it also caught your eye, I'll changing since I'm at it.

Reply:
There are other occurences of `n`. The template-argument name is visible in the [documentation](http://brunocodutra.github.io/metal/group__lambda.html#ga5adc3e3a25b8ce55e4993fa719b85ce3). Anyway, let us not get lost in details here.

Reply:
Oh boy

Reply:
I guess at this point it's easier to revert this and rename the `n`s to `m`s.

Reply:
I apologize for overcomplicating this.

What matters here is the 1-based vs. 0-based issue. Thanks for the quick clarification.

Reply:
Not at all - I do care about these little details and I'm glad you helped me get it right.